src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,12 @@ public class UserService {
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
+    public static User findByEmail(String email) {
+        for (User obj: users) {
+            if (obj.getEmail().equals(email)) {
+                return obj;
+            }
+        }
```

Comment:
Dont use static method here
```suggestion
    public User findByEmail(String email) {
        for (User obj: users) {
            if (obj.getEmail().equals(email)) {
                return obj;
            }
        }
```

Reply:
if i remove the static access modifier here i get a 'Non-static method 'findByEmail(java.lang.String)' cannot be referenced from a static context' problem in my login method. am i missing something?

Reply:
@sokimaaa ??